docs/guides/places.mdx

```diff
@@ -27,8 +28,8 @@ The Overture places theme has one feature type, called `place`, and contains mor
 |:--:|
 | *Overture places data, June 2025* |
 
-| Source |  Feature count, June 2025 release |
-| -------------------------- | --------------------- |
+| source | feature count, june 2025 |
+| --- | --- |
 | meta | 56,677,393 |
 | Microsoft | 4,655,498 |
 | Microsoft & meta | 2,891,357 |
```

Comment:
Not directly related to your changes, but I think we should probably alphabetize the ordering of these combined sources so they are consistently written out.

Reply:
Good call, these are now alphabetized